src/tcgwars/logic/impl/gen3/Deoxys.groovy

```diff
@@ -547,8 +567,10 @@ public enum Deoxys implements LogicCardInfo {
             attackRequirement {}
             onAttack {
               damage 30
-              if(opp.hand.size() >= 5){
-                opp.hand.oppSelect(max : opp.hand.size() - 4,min : opp.hand.size() - 4,"Select the card to discard").discard()
+              afterDamage {
+                while (opp.hand.size() >= 5){
```

Comment:
I think changing this to a while loop misrepresents what it's doing. There's no way to actually make this loop as with count you _have_ to choose that number of cards, do you not?

Reply:
Oh, actually thought of replacing it with a single card discard each time... And then thought it didn't add anything meaningful to anyone so I undid it, but left the while by mistake lmao

Will correct in a bit, good catch!